subscriptions/tasks.py

```diff
@@ -675,6 +675,35 @@ def run(self):
 fire_week_estimate_last = FireWeekEstimateLast()
 
 
+class FireDailySendEstimate(Task):
+    """Fires daily estimated send counts.
+    """
+    name = "subscriptions.tasks.fire_daily_send_estimate"
+
+    def run(self):
+        # Django's datetime's weekday method has Monday = 0
+        # whereas the cron format used in the schedules has Sunday = 0
+        day = now().weekday() + 1
+
+        schedules = Schedule.objects.filter(
+            Q(day_of_week__contains=day) | Q(day_of_week__contains='*'),
+            subscriptions__active=True,
+            subscriptions__completed=False,
+            subscriptions__process_status=0
+        ).values('subscriptions__messageset').annotate(
+            total_subs=Count('subscriptions'))
+
+        for schedule in schedules:
+            EstimatedSend.objects.get_or_create(
+                send_date=now().date(),
+                messageset_id=schedule['subscriptions__messageset'],
+                estimate=schedule['total_subs']
+            )
```

Comment:
So I think this will help us to answer the question "What percentage of subscriptions had successful message sends", but I don't think that it will be able to answer the question "What percentage of mothers had successful message sends", since a mother can have multiple subscriptions.

Maybe we should have two counts on the model, the number of subscriptions that we estimate will run on that day, and the number of subscriptions unique on the identity UUID that we estimate will run on that day.